.gitignore

```diff
@@ -15,6 +15,7 @@ pnpm-debug.log*
 
 # Editor directories and files
 .idea
+.vscode
```

Comment:
Why remove all of vscode recommended settings and so on?

Reply:
didn't realize there were specific settings tbh, I'm just used to gitignoring the IDE settings files. I'll update the PR and take that back out :+1: